compiler/src/iree/compiler/Dialect/LinalgExt/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -2065,6 +2086,19 @@ OnlineAttentionOp::getTiledImplementation(OpBuilder &builder,
 
   tiledOperands.emplace_back(scale);
 
+  // Mask
+  Value attnMask = getMask();
+  if (attnMask != nullptr) {
```

Comment:
```suggestion
  if (Value attnMask = getMask()) {
```

Reply:
This would pass even when attnMask is <<NULL_VALUE>>

Reply:
What is `<<NULL_VALUE>>`? This sounds like a symptom of some other bug.

Reply:
I'll check, but iirc It's a consequence of setting mask to Value() when not inputted (in one of the build functions)

Reply:
`bool(Value())` should be false and fail this check

Reply:
🤯 What sorcery! It works now